doc-content/shared-kie-docs/src/main/asciidoc/Workbench/RemoteAPI/knowledge-store-rest-api-projects-ref.adoc

```diff
@@ -0,0 +1,439 @@
+  "publicURIs": [
+    {
+      "protocol": "git",
+      "uri": "git://localhost:9418/MySpace/example-Employee_Rostering"
+    },
+    {
+      "protocol": "ssh",
+      "uri": "ssh://localhost:8001/MySpace/example-Employee_Rostering"
+    }
+  ]
+}
+----
+--
+[POST] /spaces/{spaceName}/projects::
++
+--
+Creates a project in a specified space.
+
+.Request parameters
+[cols="25%,45%,15%,15%", frame="all", options="header"]
+|===
+|Name
+|Description
+|Type
+|Requirement
+
+|`spaceName`
+|Name of the space in which the new project will be created
+|String
+|Required
+
+|*body*
+|The `name`, `groupId`, `version`, `description`, and any other components of the new project
+|Request body
+|Required
+|===
+
+.Example request body (JSON)
+[source,json]
+----
+{
+  "name": "Employee_Rostering",
+  "groupId": "employeerostering",
+  "version": "1.0.0-SNAPSHOT",
+  "description": "Employee rostering problem optimisation using Planner. Assigns employees to shifts based on their skill."
+}
+----
+
+.Example server response (JSON)
+[source,json]
+----
+{
+  "jobId": "1541017411591-6",
+  "status": "APPROVED",
+  "spaceName": "MySpace",
+  "projectName": "Employee_Rostering",
+  "projectGroupId": "employeerostering",
+  "projectVersion": "1.0.0-SNAPSHOT",
+  "description": "Employee rostering problem optimisation using Planner. Assigns employees to shifts based on their skill."
+}
+----
+--
+[DELETE] /spaces/{spaceName}/projects/{projectName}::
++
+--
+Deletes a specified project from a specified space.
+
+.Request parameters
+[cols="25%,45%,15%,15%", frame="all", options="header"]
+|===
+|Name
+|Description
+|Type
+|Requirement
+
+|`spaceName`
+|Name of the space where the project resides
+|String
+|Required
+
+|`projectName`
+|Name of the project to be deleted
+|String
+|Required
+|===
+
+.Example server response (JSON)
+[source,json]
+----
+{
+  "jobId": "1541128617727-10",
+  "status": "APPROVED",
+  "projectName": "Employee_Rostering",
+  "spaceName": "MySpace"
+}
+----
+--
+[POST] /spaces/{spaceName}/git/clone::
++
+--
+Clones a project into a specified space from a specified Git address.
+
+.Request parameters
+[cols="25%,45%,15%,15%", frame="all", options="header"]
+|===
+|Name
+|Description
+|Type
+|Requirement
+
+|`spaceName`
+|Name of the space to which you are cloning a project
+|String
+|Required
+
+|*body*
+|The `name`, `description`, and Git repository `userName`, `password`, and `gitURL` for the project to be cloned
+|Request body
+|Required
+|===
+
+*<@Alex: This request is successful and returns the response show, but the cloned `Employee_Rostering` project doesn't appear in the list of projects in the new space to which I've specified to clone it as I'd expect. This request contains the name, git URL, and git login info for the project in `MySpace` that I'm specifying to clone into `MySpace2`, but I can't get it to actually show up in `MySpace2`. Am I missing something or misunderstanding what this does?>*
```

Comment:
I'll let someone from QE, that uses this API quite intensively, to make a comment.

@jhrcek what would be the best contact to help with this?

Reply:
Thanks @jhrcek. You seem to be using the same components in my REST request example, so maybe it was just something else I missed. I'll look into it. Thanks!